scavenger-collector/build.gradle.kts

```diff
@@ -37,20 +37,20 @@ dependencies {
     implementation("com.navercorp.spring:spring-boot-starter-data-jdbc-plus-sql:${property("springDataJdbcPlusVersion")}")
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
     implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310") // test
-    implementation("mysql:mysql-connector-java")
+    implementation("com.mysql:mysql-connector-j")
     implementation("com.github.ben-manes.caffeine:caffeine")
     implementation("com.h2database:h2:2.1.212")
     implementation("org.liquibase:liquibase-core")
     implementation("io.grpc:grpc-kotlin-stub:${property("grpcKotlinVersion")}")
-    implementation("com.linecorp.armeria:armeria-spring-boot2-starter")
-    implementation("com.linecorp.armeria:armeria-tomcat9")
+    implementation("com.linecorp.armeria:armeria-spring-boot3-starter")
+    implementation("com.linecorp.armeria:armeria-tomcat10")
     implementation("com.linecorp.armeria:armeria-grpc")
     implementation("io.github.microutils:kotlin-logging-jvm:2.1.23")
     implementation("net.ttddyy:datasource-proxy:1.7")
     implementation("com.github.vertical-blank:sql-formatter:1.0.1")
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("org.mockito.kotlin:mockito-kotlin:3.2.0")
```

Comment:
It is using mockito-core 5.7.0 on springboot 3.2.4.
Changing to https://mvnrepository.com/artifact/org.mockito.kotlin/mockito-kotlin/5.3.1

Reply:
fixed in https://github.com/naver/scavenger/pull/124/commits/576081e870da04e39edc820836090269dcec0a4a